services/management/grpc/actions_server.go

```diff
@@ -294,6 +295,60 @@ func (s *actionsServer) StartPTSummaryAction(ctx context.Context, req *managemen
 	}, nil
 }
 
+func pointerToAgentType(agentType models.AgentType) *models.AgentType {
+	return &agentType
+}
+
+// StartPTMySQLSummaryAction starts pt-mysql-summary action and returns the pointer to the response message
+//nolint:lll
+func (s *actionsServer) StartPTMySQLSummaryAction(ctx context.Context, req *managementpb.StartPTMySQLSummaryActionRequest) (*managementpb.StartPTMySQLSummaryActionResponse, error) {
+	// Need to get the service id's pointer to retrieve the list of agent pointers therefrom
+	// to get the particular agentID from the request.
+	service, err := models.FindServiceByID(s.db.Querier, req.ServiceId)
+	if err != nil {
+		return nil, err
+	}
+
+	agents, err := models.FindPMMAgentsRunningOnNode(s.db.Querier, service.NodeID)
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "No pmm-agent running on this node")
+	}
+
+	agentID, err := models.FindPmmAgentIDToRunAction(req.PmmAgentId, agents)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := models.CreateActionResult(s.db.Querier, agentID)
+	if err != nil {
+		return nil, err
+	}
+
+	// Need to get the mysql exporters to get the username and password therefrom
+	psExporters, err := models.FindAgents(s.db.Querier, models.AgentFilters{AgentType: pointerToAgentType(models.MySQLdExporterType)})
```

Comment:
Could you tell us your understanding of consistency?
What is the reason of usage new naming style? what does `ps` means and what does it bring to this place?

Reply:
Also this logic is wrong. This request will return all MySQLd exporters for all nodes.

Reply:
"Could you tell us your understanding of consistency?" Can you please be more specific or rephrase this?
I haven't found any rule dedicated to pointers, variables and structures naming therefore I'm not aware of having broken any rule. If there is a variables and pointers, variables, structures naming rule, please send me a link to it and the rule number.

Reply:
We need consistency with other code in this file, package, repository, project.

We don't use Hungarian notation in PMM, so please do not introduce it.

Reply:
Roger that.